docs/ha-considerations.md

```diff
@@ -91,15 +91,24 @@ errorExit() {
     exit 1
 }
 
-curl --silent --max-time 2 --insecure https://localhost:${APISERVER_DEST_PORT}/ -o /dev/null || errorExit "Error GET https://localhost:${APISERVER_DEST_PORT}/"
-if ip addr | grep -q ${APISERVER_VIP}; then
-    curl --silent --max-time 2 --insecure https://${APISERVER_VIP}:${APISERVER_DEST_PORT}/ -o /dev/null || errorExit "Error GET https://${APISERVER_VIP}:${APISERVER_DEST_PORT}/"
+# replace this with the virtual IP address negotiated between the `keepalived` cluster hosts
+APISERVER_VIP="10.88.111.88"
```

Comment:
we shouldn't define these vars here

Reply:
It's defined so that it can be reused in the script. If you mean the value of `"10.88.111.88"`, I changed the code to put `${APISERVER_VIP_VALUE}` there so that we don't have a random value there.